recipes/sip/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    pythonw configure.py --sysroot=$PREFIX
```

Comment:
Still pretty handwavy, but this is a bit better [answer]( http://stackoverflow.com/a/30313091 ).